Sources/MapboxNavigation/CarPlayManagerDelegate.swift

```diff
@@ -53,6 +56,8 @@ public protocol CarPlayManagerDelegate: AnyObject, UnimplementedLogging {
      */
     func carPlayManager(_ carPlayManager: CarPlayManager, mapButtonsCompatibleWith traitCollection: UITraitCollection, in carPlayTemplate: CPTemplate, for activity: CarPlayActivity) -> [CPMapButton]?
     
+    // MARK: Navigation Events Handling
```

Comment:
“Navigation events” sounds too close to NavigationEventsManager. I think this is essentially what we often refer to as “user location tracking” or “route tracking”.